### PR TITLE
fix ConfigConvert[Option[_]].to for None

### PR DESCRIPTION
**Pull request by melrief**
Doing this before #151 because I'd like to have a commit in the git history

### CODE REVIEW

**Review by jcazevedo — APPROVED** [review]

File: core/src/test/scala/pureconfig/PureconfSuite.scala [hunk 1]
```diff
@@ -51,6 +51,11 @@ class PureconfSuite extends FlatSpec with Matchers with OptionValues with Either
     }
   }
 
+  "ConfigConvert" should s"be able to convert from and to None" in {
```
Comment (derekmorr) [inline core/src/test/scala/pureconfig/PureconfSuite.scala]: minor nit: We shouldn't need the "s" here, but it's harmless to leave it in.

**Comment by derekmorr** [comment]
Looks good to me. I made one very small comment, which you can feel free to ignore.

File: core/src/test/scala/pureconfig/PureconfSuite.scala (hunk 1 above)
Comment (melrief) [reply core/src/test/scala/pureconfig/PureconfSuite.scala]: Fixed, thanks for pointing out when I add the `s` without reason. I think I'm doing it a lot and it's completely involuntary.

**Comment by melrief** [comment]
Ok then, I think we can merge. @derekmorr in the comments for PRs there is a button to approve or request changes for a PR, if you want. It's easier for the contributor to see the list of reviewers and what they think in that way because it's just on the right of the PR when you open it.

**Review by derekmorr — APPROVED** [review]

**Pull request merged**